src/components/ItemFilter/FilterSettingsDialog.js

```diff
@@ -0,0 +1,202 @@
+            <Radio
+                dense
+                checked={filtersSelectable}
+                label={i18n.t('Only allow filtering on specified dimensions')}
+                name="radio-restrict-filtering"
+                onChange={payload => {
+                    return updateFilterDimensionRestrictability(
+                        payload.value === 'true'
+                    )
+                }}
+                value={'true'}
+            />
+        </div>
+    </form>
+)
+
+RadioOptions.propTypes = {
+    filtersSelectable: PropTypes.bool,
+    updateFilterDimensionRestrictability: PropTypes.func,
+}
+
+const FilterSettingsDialog = ({
+    dimensions,
+    restrictFilters,
+    initiallySelectedItems,
+    onClose,
+    onConfirm,
+    open,
+}) => {
+    const [selected, setSelected] = useState(initiallySelectedItems)
+    const [filtersSelectable, setFiltersSelectable] = useState(restrictFilters)
+
+    const updateFilterDimensionRestrictability = val => {
+        if (val) {
+            const otherItems = selected.filter(
+                i => i !== DIMENSION_ID_ORGUNIT && i !== DIMENSION_ID_PERIOD
+            )
+            updateSelectedFilters([
+                DIMENSION_ID_PERIOD,
+                DIMENSION_ID_ORGUNIT,
+                ...otherItems,
+            ])
+        }
+        setFiltersSelectable(val)
+    }
+
+    const updateSelectedFilters = sItems => {
+        // ensure that orgUnit, period appear before other items
+        const periodItem = [
+            ...new Set(sItems.filter(i => i === DIMENSION_ID_PERIOD)),
+        ]
+        const orgUnitItem = [
+            ...new Set(sItems.filter(i => i === DIMENSION_ID_ORGUNIT)),
+        ]
+        const otherItems = sItems.filter(
+            i => i !== DIMENSION_ID_ORGUNIT && i !== DIMENSION_ID_PERIOD
+        )
+        return setSelected([...periodItem, ...orgUnitItem, ...otherItems])
+    }
+
+    const closeDialog = () => {
+        setSelected(initiallySelectedItems)
+        setFiltersSelectable(restrictFilters)
+        onClose()
+    }
+
+    return (
+        <>
+            {open && (
+                <Modal
+                    dataTest="dhis2-uicore-modal"
+                    large
+                    position="top"
+                    onClose={closeDialog}
+                >
+                    <ModalTitle>
+                        {i18n.t('Dashboard Filter Settings')}
+                    </ModalTitle>
+                    <ModalContent>
+                        <div className={classes.descContainer}>
+                            <span>
+                                {i18n.t(`Filters allow users of a dashboard to customize the 
+                                data displayed. By default all dimensions are 
+                                available as filters. Alternatively, a limited 
+                                number of filters can be made available on a dashboard.`)}
+                            </span>
```

Comment:
I suggest updating this text with a slightly refined version: `Dashboards can be filtered by dimensions to change the data shown. By default, all dimensions are available as filters. Alternatively, only selected dimensions can be made available on a dashboard.`

Reply:
fixed